src/main/resources/assets/pipez/lang/de_de.json

```diff
@@ -0,0 +1,85 @@
+{
+  "itemGroup.pipez": "Röhrez",
+  "block.pipez.item_pipe": "Item Röhre",
+  "block.pipez.fluid_pipe": "Flüssigkeits Röhre",
+  "block.pipez.energy_pipe": "Energie Röhre",
+  "block.pipez.universal_pipe": "Universele Röhre",
+  "block.pipez.gas_pipe": "Gas Röhre",
+  "item.pipez.basic_upgrade": "Normale Röhre Verbesserung",
+  "item.pipez.improved_upgrade": "Verbesserte Röhre Verbesserung",
+  "item.pipez.advanced_upgrade": "Erweiterte Röhre Verbesserung",
+  "item.pipez.ultimate_upgrade": "Ultimative Röhre Verbesserung",
+  "item.pipez.infinity_upgrade": "Unendliche Röhre Verbesserung",
+  "item.pipez.wrench": "Röhren Zange",
+  "item.pipez.filter_destination_tool": "Filter Ziel Gerät",
+  "tooltip.pipez.wrench": "Klicke auf eine Röhre um sie einzustellen",
+  "tooltip.pipez.distribution": "Verteilung: %s",
+  "tooltip.pipez.distribution.nearest": "Nächste zuerst",
+  "tooltip.pipez.distribution.furthest": "Weiteste zuert",
+  "tooltip.pipez.distribution.round_robin": "Rundlauf",
+  "tooltip.pipez.distribution.random": "Zufällig",
+  "tooltip.pipez.redstone_mode": "Rotstein Modus: %s",
+  "tooltip.pipez.redstone_mode.ignored": "Ignoriert",
+  "tooltip.pipez.redstone_mode.off_when_powered": "Aus Wenn Aktiviert",
+  "tooltip.pipez.redstone_mode.on_when_powered": "An Wenn Aktiviert",
+  "tooltip.pipez.redstone_mode.always_off": "Immer Aus",
+  "tooltip.pipez.filter_mode": "Filter Modus: %s",
+  "tooltip.pipez.filter_mode.whitelist": "Erlaubt",
+  "tooltip.pipez.filter_mode.blacklist": "Nicht Erlaubt",
+  "tooltip.pipez.filter.not": "Nicht",
+  "tooltip.pipez.filter.accepts_tag": "Alle %s",
+  "tooltip.pipez.filter.item_tag.description": "Tags Präfix mit # oder item IDs",
+  "tooltip.pipez.filter.nbt_string.description": "NBT Data",
+  "tooltip.pipez.filter.nbt_string.no_nbt": "Dieser Typ kan keine NBT tags haben",
+  "tooltip.pipez.filter.nbt.exact": "Genaue übereinstimmung NBT data",
+  "tooltip.pipez.filter.nbt.not_exact": "Nur bereitgestellte Tags abgleichen ",
+  "tooltip.pipez.filter.not_inverted": "Regulär",
+  "tooltip.pipez.filter.inverted": "Invertiert",
+  "tooltip.pipez.filter_destination_tool": "Klicke auf einen Block, um das Ziel festzulegen",
+  "tooltip.pipez.filter_destination_tool.destination": "Ziel: X=%s, Y=%s, Z=%s, %s",
+  "tooltip.pipez.filter.destination.description": "Benutze das Filter Ziel Gerät um ein Ziel festzulegen",
+  "tooltip.pipez.filter.destination.click_to_remove": "Klicken zum entfernen",
+  "tooltip.pipez.filter.unknown_block": "Unbekannter BLOCK",
```

Comment:
Why "BLOCK"?

Reply:
it was written all cap in Original. Will change